cmd/antrea-controller/config.go

```diff
@@ -45,4 +45,19 @@ type ControllerConfig struct {
 	TLSMinVersion string `yaml:"tlsMinVersion,omitempty"`
 	// Legacy CRD mirroring.
 	LegacyCRDMirroring bool `yaml:"legacyCRDMirroring,omitempty"`
+	// Enable the integrated node IPAM controller within the Antrea controller.
+	// Defaults to false.
+	EnableNodeIPAMController bool `yaml:"enableNodeIPAMController,omitempty"`
+	// CIDR Ranges for Pods in cluster. Value can contain a single CIDR range, or multiple ranges, separated by commas.
+	// The CIDRs could be either IPv4 or IPv6. Value ignored when EnableNodeIPAMController is false.
+	ClusterCIDRs string `yaml:"clusterCIDRs,omitempty"`
+	// CIDR Range for Services in cluster. It is not necessary to specify it when there is no overlap with clusterCIDRs.
+	// Value ignored when EnableNodeIPAMController is false.
+	ServiceCIDR string `yaml:"serviceCIDR,omitempty"`
```

Comment:
If `ClusterCIDRs` can have multiple ranges and can contain either IPv4 or IPv6, how to exclude an IPv4 and an IPv6 service CIDR in dual stack environment if serviceCIDR takes only one CIDR? 
Kubernetes has `SecondaryServiceCIDR` option for it. We should either follow it or change it to `serviceCIDRs`

Reply:
At first glance it might be better to have a SecondaryServiceCIDR. Since there should be at most 1 v4 and 1 v6 cluster IP CIDRs.

Reply:
Done